aten/src/ATen/cpu/vec/vec256/vec256_complex_double.h

```diff
@@ -367,17 +367,24 @@ template <> Vectorized<c10::complex<double>> inline operator*(const Vectorized<c
 
 template <> Vectorized<c10::complex<double>> inline operator/(const Vectorized<c10::complex<double>> &a, const Vectorized<c10::complex<double>> &b) {
   //re + im*i = (a + bi)  / (c + di)
-  //re = (ac + bd)/abs_2()
-  //im = (bc - ad)/abs_2()
-  const __m256d sign_mask = _mm256_setr_pd(-0.0, 0.0, -0.0, 0.0);
-  auto ac_bd = _mm256_mul_pd(a, b);         //ac       bd
+  auto mask = _mm256_set1_pd(-0.f);
+  auto fabs_cd = _mm256_andnot_pd(mask, b);     // |c|    |d|
+  auto fabs_dc = _mm256_permute_pd(fabs_cd, 0x05);   // |d|    |c|
+  auto scale = _mm256_max_pd(fabs_cd, fabs_dc);  // sc     sc
+  auto a2 = _mm256_div_pd(a, scale);         // a/sc     b/sc
+  auto b2 = _mm256_div_pd(b, scale);         // c/sc     d/sc
```

Comment:
can you try using `scale = 1/max` here as well, but just using div instead of rcp?

Reply:
I can try it